### ISSUE TITLE
GitLab Webhooks fail

### ISSUE THREAD

**Issue by Bonscho** [terramar-labs/packages#34]
Using GitLab 8.3.2 all webhooks fail with message `Hook execution failed: Accepted`

**Comment by Bonscho** [comment]
**Additional info**  
The command line satis:build fails with 
```
[Composer\Json\JsonValidationException]                       
The json config file does not match the expected JSON schema 
```

**Comment by Bonscho** [comment]
satis.json is
```
{
    "output-dir": "\/var\/www\/packages\/web",
    "repositories": [
        {
            "type": "vcs",
            "url": "git@git.example.com:xxx\/xxx.git"
        },
        {
            "type": "vcs",
            "url": "git@git.example.com:xxx\/xxx-ui-kit.git"
        },
        {
            "type": "vcs",
            "url": "git@git.example.com:xxx\/xxx2.git"
        }
    ],
    "output-html": false,
    "require-dependencies": true,
    "require-dev-dependencies": true
}
```

**Comment by Bonscho** [comment]
The problem is that the satis.json is validated as composer.json. So the mandatory members `name` and `homepage` are missing. After adding these two members manually to the satis.json the `satis:build` call is successful.

**Comment by Bonscho** [comment]
I came down with this beeing two different problems.  
At first the satis.json is validated against a composer.json validator by the satis lib.  
At second GitLab does not accept HTTP status 202 a valid successful response to webhooks.  
I will PR the solution for generating a satis.json that passes the conposer.json validation. This solves the problem but for sure is not the way satis should act in the future.

**Comment by tyler-sommer** [comment]
Fixed in b73c5a3 and 8d42a31. Thanks again, @Bonscho!
